rsts/getting_started.rst

```diff
@@ -3,107 +3,324 @@
+
+           .. prompt::
+
+               docker build . --tag registry/repo:version
+               docker push registry/repo:version
+
+#. Now that the container is built, lets provide this information to the Flyte backend. To do that you have to package the workflow using the ``pyflyte`` cli, that is bundled with flytekit. Also note, the image is the same as the one built in the previous step::
+
+    pyflyte --pkgs myapp.workflows package --image myapp:v1
+
+#. Now lets upload this package to flyte backend. We refer to this as ``registration`` ::
+
+    flytectl register files -p flytesnacks -d development -a flyte-package.tgz  -v v1
+
+
+.. _getting-started-execute:
+
+Execute in remote cluster
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Use FlyteConsole to launch an execution and watch the progress.
 
 .. image:: https://raw.githubusercontent.com/flyteorg/flyte/static-resources/img/flytesnacks/tutorial/exercise.gif
     :alt: A quick visual tour for launching a workflow and checking the outputs when they're done.
 
-.. admonition:: Recap
+**Alternatively** Launch and monitor from CLI using flytectl
+
+Launch an execution using flytectl::
+
+        TODO
+
+Retrieve execution status using flytectl::
+
+        TODO
+
+
+Modify code: Modify and test locally
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+#. Open ``example.py`` in your favorite editor.
+
+   .. code-block::
+
+       myapp/workflows/example.py
+
+   .. dropdown:: myapp/workflows/example.py
+
+      .. rli:: https://raw.githubusercontent.com/flyteorg/flytekit-python-template/simplify-template/myapp/workflows/example.py
+         :language: python
+
+#. Add ``name: str`` as an argument to both ``my_wf`` and ``say_hello`` functions. Then update the body of ``say_hello`` to consume that argument.
+
+   .. code-block:: python
+
+     @task
+     def say_hello(name: str) -> str:
+         return f"hello world, {name}"
+
+   .. code-block:: python
+
+     @workflow
+     def my_wf(name: str) -> str:
+         res = say_hello(name=name)
+         return res
+
+#. Update the simple test at the bottom of the file to pass in a name. E.g.
+
+   .. code-block:: python
+
+     print(f"Running my_wf(name='adam') {my_wf(name='adam')}")
+
+#. When you run this file locally, it should output ``hello world, adam``. Run this command in your terminal:
+
+   .. prompt::
+
+     python myapp/workflows/example.py
+
+
+   .. dropdown:: Expected output
+
+       .. prompt::
+
+            Running my_wf(name='adam') hello world, adam
+
+
+Fast: Deploy your application quickly
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+#. To deploy this workflow to the Flyte cluster (sandbox), you can repeat the previously explained :ref:`getting-started-standard`. But, Flyte provides a faster way to iterate on your workflows. Since you have not really updated any of the dependencies in your requirements file, it is possible to push just the code to Flyte backend, without really re-building the entire docker container.
```

Comment:
```suggestion
#. To deploy this workflow to the Flyte cluster (sandbox), you can repeat the steps previously covered in :ref:`getting-started-standard`.  Flyte provides a faster way to iterate on your workflows. Since you have not updated any of the dependencies in your requirements file, it is possible to push just the code to Flyte backend, without having to re-build the entire docker container.
```